### DISCUSSION TITLE
v-if dans un groupe répétable qui ne fonctionne pas

### DISCUSSION THREAD

**Discussion by drogu02** [MTESSDev/FRW#185] [Annonces]
Bonjour, j'ai essayé ça et les v-if ne fonctionnent pas. Ques que je devrais faire?

- name: periodesIndemnitesGroupe
              type: repeatableGroup
              repeatable: true
              limit: 5
              label:
                fr: Informations pour la période
              instanceLabel:
                fr: Inscrire la période
              addLabel:
                fr: Ajouter une période
              removeLabel: 
                fr: Supprimer la période
              components:
                - name: nombreJoursAvantAudience
                  type: radio
                  label: 
                    fr: Nombre de jours avant la date de l'audience
                  outerClasses: boutons-inline md
                  options:
                    plusDe: 
                      fr: Plus de
                    entre: 
                      fr: Entre
                    moinsDe: 
                      fr: Moins de
                - name: nombreJoursAvantAudienceMin
                  type: number
                  v-if: val('nombreJoursAvantAudience') == 'plusDe' || val('nombreJoursAvantAudience') == 'entre'
                  label:
                    fr: Nombre de jours minimum
                  min: 0
                  validations:
                    min: 0
                    optional: null
                - name: nombreJoursAvantAudienceMax
                  type: number
                  v-if: val('nombreJoursAvantAudience') == 'moinsDe' || val('nombreJoursAvantAudience') == 'entre'
                  label:
                    fr: Nombre de jours maximum
                  min: 0
                  validations:
                    min: 0
                    optional: null

**Comment by j0ber** [comment]
Bonjour,

Il faut faire référence au groupe dans le v-if.

Voir https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionConditionSi  exemple 4.

**Comment by drogu02** [comment]
Parfait, ça fonctionne!  Merci monsieur Caux!
